build/common.props

```diff
@@ -2,12 +2,12 @@
   <PropertyGroup>
     <!-- Extensions can have independent versions and only increment when released -->
     <Version>3.0.0$(VersionSuffix)</Version>
-    <ExtensionsVersion>4.0.2$(VersionSuffix)</ExtensionsVersion> <!-- WebJobs.Extensions -->
-    <CosmosDBVersion>3.0.8$(VersionSuffix)</CosmosDBVersion>
+    <ExtensionsVersion>4.0.3$(VersionSuffix)</ExtensionsVersion> <!-- WebJobs.Extensions -->
+    <CosmosDBVersion>3.0.9$(VersionSuffix)</CosmosDBVersion>
     <HttpVersion>3.0.12$(VersionSuffix)</HttpVersion>
     <MobileAppsVersion>3.0.0$(VersionSuffix)</MobileAppsVersion>
-    <SendGridVersion>3.0.2$(VersionSuffix)</SendGridVersion>
-    <TwilioVersion>3.0.2$(VersionSuffix)</TwilioVersion>
+    <SendGridVersion>3.0.3$(VersionSuffix)</SendGridVersion>
+    <TwilioVersion>3.0.3$(VersionSuffix)</TwilioVersion>
```

Comment:
Same here -- should this be 3.0.1? https://www.nuget.org/packages/Microsoft.Azure.WebJobs.Extensions.Twilio/